samples/msgext-bot-SPUploader/README.md

```diff
@@ -83,38 +83,48 @@ Step 1: Setup bot in Service
 7. Add any necessary API permissions for downstream calls in the App registration.
 	* Navigate to "API permissions" blade on the left-hand side.
 	* Add following permissions to the application.
-		* Application permissions - User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All,
-									GroupMember.Read.All, Group.Read.All, GroupMember.ReadWrite.All, Group.ReadWrite.All,Directory.AccessAsUser.All
-		* Delegated permissions - User.Read (enabled by default)
+	* Application permissions - User.Read.All and GroupMember.Read.All
+	* Delegated permissions - User.Read (enabled by default)
 
-<img src="./Docs/Images/APIPermissions.PNG" alt="API Permissions" width="1000" height="500">
+<img src="./Docs/Images/ApiPermissions.PNG" alt="API Permissions" width="1000" height="500">
```

Comment:
This image is not loading, can you fix the image src / name

Reply:
Resolved the issue